### DISCUSSION TITLE
请教下大佬，DOCKER部署失败……

### DISCUSSION THREAD

**Discussion by themis0009** [GaiZhenbiao/ChuanhuChatGPT#640] [General]
命令是：
docker run -d --name chuanghu-chatgpt \
	-e my_api_key="sk-xxxxxxxxx" \
	-v ~/chatGPThistory:/app/history \
	-v ~/ChuanhuChatGPT/config.json:/app/config.json \
	-p 7860:7860 \
	tuchuanhuhuhu/chuanhuchatgpt:latest

服务器在墙外，所以不用api_host参数，删除。
然后启动错误日志是config.json错误
这个为什么？~_~

========删除重装了，错误日志又有点不同=============
Traceback (most recent call last):
  File "/app/ChuanhuChatbot.py", line 8, in <module>
    from modules import config
  File "/app/modules/config.py", line 30, in <module>
    with open("config.json", "r", encoding='utf-8') as f:
IsADirectoryError: [Errno 21] Is a directory: 'config.json'

**Comment by Qwerty4401** [comment]
docker上一版更新后需要在/ChuanhuChatGPT/config.json文件中配置参数，具体内容可参考[config_example.json](https://github.com/GaiZhenbiao/ChuanhuChatGPT/blob/main/config_example.json)
